docs/src/components/landing/Hero.js

```diff
@@ -196,8 +196,11 @@ export default function Hero() {
           to data sources with your own code.
         </Typography>
         <GetStartedButtons
+          primaryLabel={'Get started'}
```

Comment:
Why these properties? DO we plan to reuse `GetStartedButtons` somewhere with different labels?

Reply:
Agreed, it happened since I copied over the changes I suggested in https://github.com/mui/material-ui/pull/38283